lib/scope.js

```diff
@@ -2111,7 +2290,7 @@ module.exports = {
       if ( !link ) { await scope.addToReport(scope, { type: `error`, value: msg }); }
       expect( link, msg ).to.exist;
 
-      await link.evaluate( elem => { return elem.click(); });
+      await scope.tapElement( scope, elem );
```

Comment:
```suggestion
      await scope.tapElement( scope, link );
```
Because `elem` isn't defined in this function, so I'm assuming it's copy-pasta from somewhere else.